app/src/main/java/com/vuongnq14798/vuongmp3/MainActivity.kt

```diff
@@ -2,11 +2,15 @@ package com.vuongnq14798.vuongmp3
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import com.vuongnq14798.vuongmp3.base.BaseActivity
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : BaseActivity() {
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+    override val layoutResId: Int = R.layout.activity_main
```

Comment:
Trong Kotlin, khi mình khai báo var layoutResId: Int nó sẽ tự tạo cho mình hàm getter, setter . Nếu là val thì sẽ không có setter 
Kotlin cho phép mình abstract thuộc tính, tuy nhiên nếu viết như này, nó sẽ sinh thêm 1 thuộc tính ở lớp con `priavte val layoutResId: Int = R.layout.activity_main` và cũng override lại hàm getter, vì thế mình nên viết như này để không phải sinh thêm thuộc tính kia nữa 
```suggestion
    override val layoutResId: Int get() = R.layout.activity_main
```